ci-operator/config/openshift-assisted/assisted-chat-rag/openshift-assisted-assisted-chat-rag-main.yaml

```diff
@@ -0,0 +1,67 @@
+base_images:
+  assisted-service-mcp:
+    name: assisted-chat-img
+    namespace: edge-infrastructure
+    tag: assisted-service-mcp
+  rh-ecosystem-edge-assisted-chat-install-source:
+    name: assisted-chat-img
+    namespace: edge-infrastructure
+    tag: rh-ecosystem-edge-assisted-chat-install
+  rh-ecosystem-edge-assisted-chat-test:
+    name: assisted-chat-img
+    namespace: edge-infrastructure
+    tag: rh-ecosystem-edge-assisted-chat-test
+build_root:
+  project_image:
+    dockerfile_literal: |
+      FROM registry.access.redhat.com/ubi9/python-312:latest
+      USER 0:0
+images:
+- dockerfile_path: Containerfile
+  to: rag-source
+- dockerfile_literal: |
+    FROM RUNTIME_IMAGE
+    ARG RAG_SOURCE
+    COPY --chown=1001:1001 --from=rag:test /all-mpnet-base-v2/ /app-root/all-mpnet-base-v2/
+    COPY --chown=1001:1001 --from=rag:test /llama_stack_vector_db/ /app-root/llama_stack_vector_db/
+  from: rh-ecosystem-edge-assisted-chat-install-source
+  inputs:
+    rag-source:
+      as:
+      - rag:test
+  to: rh-ecosystem-edge-assisted-chat-install
```

Comment:
What is this actual image? Is this supposed to be the chat image with rag? Why then is it called `chat-install`? Seems like an odd name.